shells/browser/shared/src/renderer.js

```diff
@@ -10,13 +10,4 @@
 
 import { attach } from 'src/backend/renderer';
 
-Object.defineProperty(
-  window,
-  '__REACT_DEVTOOLS_ATTACH__',
-  ({
-    enumerable: false,
```

Comment:
The `enumerable: false` bit is important.

Reply:
Good call, I somehow missed that :+1: